tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -120,14 +123,15 @@ function Header() {
               {authenticated && (
                 <>
                   <NavItem pathname={pathname} tabTitle="Home" href="/home" />
+                  {/* potential issue here with assigning a group to an account with a pending status. */}
```

Comment:
see the (*) note in the description, would like to discuss this whenever possible